go-server-server/go/default.go

```diff
@@ -280,8 +280,15 @@ func ConfigInterfaceVlanPost(w http.ResponseWriter, r *http.Request) {
         return
     }
     if vlan_kv != nil {
-        WriteRequestErrorWithSubCode(w, http.StatusConflict, RESRC_EXISTS,
-              "Object already exists: " + vlan_name, []string{}, "")
+        vlan_if_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VLAN_INTF_TB, vlan_name))
+        if vlan_if_kv != nil {
+            vnet_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VNET_TB, vlan_if_kv["vnet_name"]))
```

Comment:
can we check if vnet_name exists in the vlan_if_kv table. It is possible that some stale entry without vnet_name may exists .